### PR TITLE
Change ordering of includes to create a more natural reading order

### PR DESCRIPTION
**Pull request by lulf**
* Fix attribute replacement
* Reuse snippets for creating address space and addresses
* Use downloadable example files.

### CODE REVIEW

File: documentation/assemblies/assembly-monitoring-kube.adoc [hunk 1]
```diff
@@ -11,9 +11,9 @@
 
 You can use Prometheus and Grafana for monitoring the {ProductName} service.
 
-include::../modules/proc-deploy-prometheus.adoc[]
+include::../modules/proc-deploy-prometheus.adoc[leveloffset=+1]
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-monitoring-kube.adoc]: I think I fixed this in my very last commit, for both files

File: documentation/modules/proc-create-address-space-cli.adoc [hunk 2]
```diff
@@ -8,30 +8,33 @@
...
+[options="nowrap",subs="attributes"]
 ----
-{cmdcli} create -f space.yaml
+{cmdcli} create -f standard-address-space.yaml
 ----
++
+The address space is ready for use when the field `.status.isReady` is set to `true`.
 
-. List the address spaces:
+. Check status of address space:
```
Comment (jenmalloy) [inline documentation/modules/proc-create-address-space-cli.adoc]: Check the status of the address space:

File: documentation/modules/proc-create-address-space-cli.adoc [hunk 3]
```diff
@@ -8,30 +8,33 @@
...
 . Create the address space:
 +
-[source,yaml,options="nowrap"]
+[options="nowrap",subs="attributes"]
 ----
-{cmdcli} create -f space.yaml
+{cmdcli} create -f standard-address-space.yaml
 ----
++
+The address space is ready for use when the field `.status.isReady` is set to `true`.
```
Comment (jenmalloy) [inline documentation/modules/proc-create-address-space-cli.adoc]: when the `.status.isReady` field is

**Comment by lulf** [comment]
@jenmalloy Updated with fixes from your comment

**Pull request merged**